tests/lib/rules/array-bracket-spacing.js

```diff
@@ -618,13 +617,13 @@ ruleTester.run("array-bracket-spacing", rule, {
                 column: 6,
                 endLine: 1,
                 endColumn: 7
-            }]
+            }],
+            languageOptions: { ecmaVersion: 6 }
```

Comment:
eslint-plugin-eslint-plugin enforces a certain order for test properties, so a lot of these files have order changes caused by `--fix`. I'm guessing these files hadn't been modified in a while so they never had lint fixes applied in this way.

Reply:
@mdjermanovic 

> What do you think about fixing the order in this PR (diff would be smaller) or at some later point?

Sorry, I'm not clear on what you're asking here. Are you saying fixing the order by configuring the rule differently? Manually fixing the order so `languageOptions` comes before `errors`?

Reply:
Sorry for not being clear.

I think it would be best to configure the rule differently (as in https://github.com/eslint/eslint/pull/17819#discussion_r1422591180) and then run `eslint . --fix` to update the tests, in this PR.

Reply:
Got it. 👍